www/simple.php

```diff
@@ -253,9 +253,13 @@ function getScheme() {
 	$hubic_user = $_GET['hubic_user'];
 	$hubic_password = $_GET['hubic_password'];
 
+        if (is_array($clients[$client]) && array_key_exists('autoredirect_uri', $clients[$client])) {
```

Comment:
Just as little remark: You could use `if (isset($clients[$client]['autoredirect_uri']) ` here. That's what I teach people in that case, because I think it looks clearer and lets you copy the code for check and usage.